networking/networking.go

```diff
@@ -229,18 +233,28 @@ func Load(podRoot string, podID *types.UUID) (*Networking, error) {
 	}, nil
 }
 
-func (n *Networking) GetDefaultIP() net.IP {
+func (n *Networking) GetDefaultIP() (net.IP, error) {
```

Comment:
Agreed, I'm also going to add this to the networking documentation as this is not very intuitive.